docs/_reporting/hubs/configure-scopes.md

```diff
@@ -201,15 +201,8 @@ If you cannot grant permissions for your scope, you can create Cost Management e
    - **Overwrite data** = Off<sup>4</sup>
    - **Storage account** = (Use subscription/resource deployed with your hub)
    - **Container** = `msexports`
-   - **Directory** = (Use the resource ID of the scope<sup>5</sup> you're exporting without the first "/")
-     - _**Billing account:** `providers/Microsoft.Billing/billingAccounts/{billingAccountId}`_
-     - _**Billing profile:** `providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}`_
-     - _**Department:** `providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}`_
-
-    <blockquote class="tip" markdown="1">
-      _If you get an error about invalid characters for a billing scope, replace the unsupported characters (e.g., `:`) in the billing account ID with a dash (`-`)._
-    </blockquote>
-
+   - **Directory** = (Specify the storage container and directory path<sup>5</sup> for the export file)
```

Comment:
The container shouldn't be in the path and we should stress that it be unique. There could be data ingestion issues if they use the same path for 2 exports. I'd rather keep the suggestions, but we can simplify them. We can probably not mention the MCA billing account ID, which is the only problem.
```suggestion
   - **Directory** = (Specify a unique path for this scope<sup>5</sup>)
     - _**EA billing account:** `billingAccounts/{enrollment-number}`_
     - _**MCA billing profile:** `billingProfiles/{billing-profile-id}`_
     - _**Subscription:** `subscriptions/{subscription-id}`_
     - _**Resource group:** `subscriptions/{subscription-id}/resourceGroups/{rg-name}`_
```